latch_cli/services/deprecated/rm.py

```diff
@@ -34,12 +36,27 @@ def rm(remote_path: str):
             that doesn't exist.
     """
     token = retrieve_or_login()
-    remote_path = _normalize_remote_path(remote_path)
 
-    data = {"filename": remote_path, "ws_account_id": current_workspace()}
-    headers = {"Authorization": f"Bearer {token}"}
-    response = requests.post(config.api.data.remove, headers=headers, json=data)
-
-    data = response.json()
-    if not data["success"]:
-        raise ValueError(data["error"]["data"]["message"])
+    to_remove = [remote_path] if no_glob else expand_pattern(remote_path)
+    if len(to_remove) == 0:
+        click.echo(
+            f"Could not find any files that match pattern {remote_path}",
+        )
+        return
+
+    files = "\n".join(to_remove)
```

Comment:
prefer to just do `click.confirm("Remove [# of files] files?")` by default and add a verbose option to print out every file name